### PR TITLE
Remove a request for pull request review command

### PR DESCRIPTION
**Pull request by europ**
### Dependent on: https://github.com/octokit/octokit.rb/pull/990

<br>

ManageIQ Bot will support a new command `remove_reviewer`. This command provides a "remove request for pull request review" functionality. The usage of this command requires to meet a condition - the user must be a requested reviewer in the pull request. If condition is met, the command perform an action which removes the user from the reviewer list in the pull request.  This command is the opposite of `add_reviewer` which is described in https://github.com/ManageIQ/miq_bot/pull/408

Example: `@miq-bot remove_reviewer @europ`

**NOTE**: Command with multiple reviewers is not supported

\cc @skateman 
\cc @romanblanco

### CODE REVIEW

**Comment by Fryguy** [comment]
LGTM!  Please add a spec as well.

File: spec/lib/github_service/commands/remove_reviewer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe GithubService::Commands::RemoveReviewer do
+  subject { described_class.new(issue) }
+  let(:issue) { double(:fq_repo_name => "org/repo") }
+  let(:command_issuer) { "nickname" }
+
+  before do
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "listed_user") { true }
```
Comment (skateman) [inline spec/lib/github_service/commands/remove_reviewer_spec.rb]: @europ what does the `{ true }` block at the end do here?

File: spec/lib/github_service/commands/remove_reviewer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "good_user") { true }
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "bad_user") { false }
+
+    allow(subject).to receive(:requested_reviewers).and_return(["listed_user"])
+  end
+
+  after do
+    subject.execute!(:issuer => command_issuer, :value => command_value)
+  end
```
Comment (skateman) [inline spec/lib/github_service/commands/remove_reviewer_spec.rb]: Why is this necessary? What are you cleaning up here?

File: spec/lib/github_service/commands/remove_reviewer_spec.rb (hunk 1 above)
Comment (europ) [reply spec/lib/github_service/commands/remove_reviewer_spec.rb]: @skateman .and_return(true)

File: spec/lib/github_service/commands/remove_reviewer_spec.rb (hunk 2 above)
Comment (europ) [reply spec/lib/github_service/commands/remove_reviewer_spec.rb]: It is used also in [assign_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/assign_spec.rb#L14), [add_reviewer_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/add_reviewer_spec.rb#L14), [add_label_spec](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/add_label_spec.rb#L18), [remove_label_spec.rb](https://github.com/ManageIQ/miq_bot/blob/master/spec/lib/github_service/commands/remove_label_spec.rb#L10). I took inspiration from these mentioned specs before and on this basis I created my remove_reviewer_spec.rb.

Comment (Fryguy) [reply spec/lib/github_service/commands/remove_reviewer_spec.rb]: It's weird, but all of the others do their thing-under-test in an after block.  I don't like it, but for this PR, it's consistent.

**Comment by europ** [comment]
@Fryguy [this](https://github.com/ManageIQ/miq_bot/pull/411/files#diff-7c743e8ae1171ab8956f1b6d6b4ff439R18) will not work without https://github.com/octokit/octokit.rb/pull/990 which is not merged yet.
